src/magic_folder/snapshot.py

```diff
@@ -253,9 +257,11 @@ def verify_snapshot_signature(remote_author, alleged_signature, content_capabili
     # See comments about "data_to_sign" in sign_snapshot
     data_to_verify = (
         u"{content_capability}\n"
+        u"{metadata_capability}\n"
         u"{name}\n"
     ).format(
         content_capability=content_capability,
+        metadata_capability=metadata_capability,
         name=snapshot_name,
     )
```

Comment:
Maybe the implementation could be refactored so we don't have two duplicate bits of code constructing the string that was signed.